build.gradle

```diff
@@ -64,16 +95,15 @@ compileTestKotlin {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:3.2.0')
+    api platform('com.exactpro.th2:bom:4.1.0')
 
-    implementation 'com.exactpro.th2:common:4.0.0-th2-2150-books-pages-2629057923-SNAPSHOT'
-    implementation "com.exactpro.th2:sailfish-utils:3.8.0"
-
-    implementation "org.slf4j:slf4j-log4j12"
-    implementation "org.slf4j:slf4j-api"
+    implementation 'com.exactpro.th2:common:5.1.0-dev-version-5-4085018593-8adee33-SNAPSHOT'
+    implementation "com.exactpro.th2:sailfish-utils:3.14.0"
```

Comment:
```suggestion
    implementation "com.exactpro.th2:sailfish-utils:0.0.1-book-and-page-3836172629-SNAPSHOT"
```

Reply:
There is no such version for sailfish utils. It is probably common-utils versions